openfisca_us/parameters/gov/irs/rebates/high_efficiency_electric_home_rebate_program.yaml

```diff
@@ -0,0 +1,14 @@
+description: The US caps high-efficiency home program rebats at this amount per year for appliance and non-appliance upgrades.
```

Comment:
Move these files to `gov/energy/high_efficiency_electric_home_rebate/` since it's managed by the Department of Energy (p. 583)

Name this file `total.yaml`